### ISSUE TITLE
Enable generating code for referenced external DLLs

### ISSUE THREAD

**Issue by patrykbuzowicz** [frhagn/Typewriter#171]
Would it be possible for the Settings object to also configure using the DLLs that are not projects under current SLN but instead are referenced by the project and contain some models?
We're sharing some code between different codebases and the models are also included. And it would be great to be able to have matching TS models automatically generated...

**Comment by robertmclaws** [comment]
This is even more important when you're using libraries like NodaTime for your models, and want the information to come over the wire properly.

**Comment by andrewb273** [comment]
So keen for this. We're trying to move more of our shared code into internal nuget packages, but we don't want that compromising our speed of client-side coding.

**Comment by grungondola** [comment]
I agree that this is needed. I have multiple external projects that have re-useable models for multiple projects, and it would be great to just be able to include the models from the referenced library in my Typewriter files.
